spec/views/ems_physical_infra/_form_fields.html.haml_spec.rb

```diff
@@ -0,0 +1,15 @@
+describe "rendering fields in ems_infra new/edit form" do
+  before(:each) do
+    @edit = {:new => {:emstype => "vm_ware"}}
```

Comment:
It must also be a typo in `ems_infra` specs as well?  

https://github.com/ManageIQ/manageiq-ui-classic/blob/50afa1e32b5285b52d2635e95208494a3c613f50/spec/views/ems_infra/_form_fields.html.haml_spec.rb#L3